packages/create-platformatic/test/service/create-service.test.mjs

```diff
@@ -52,11 +85,11 @@ test('creates service with no typescript', async ({ end, equal, same, ok }) => {
   equal(process.env.PORT, '6666')
 
   same(plugin, ['./plugins', './routes'])
-  ok(isFileAccessible(join(tmpDir, 'plugins', 'examples.js')))
-  ok(isFileAccessible(join(tmpDir, 'routes', 'root.js')))
+  ok(await isFileAccessible(join(tmpDir, 'plugins', 'example.js')))
+  ok(await isFileAccessible(join(tmpDir, 'routes', 'root.js')))
```

Comment:
I found we had wrong assertions. It was always passed since `isFileAccessible` returns Promise and its truthy.